docs/content/docs/deployment/memory/network_mem_tuning.md

```diff
@@ -120,6 +120,19 @@ In order to avoid excessive data skew, the number of buffers for each subpartiti
 
 Unlike the input buffer pool, the configured amount of exclusive buffers and floating buffers is only treated as recommended values. If there are not enough buffers available, Flink can make progress with only a single exclusive buffer per output subpartition and zero floating buffers.
 
+#### Overdraft buffers
+
+For each output subtask can also request up to `taskmanager.network.memory.max-overdraft-buffers-per-gate` (by default 5) extra overdraft buffers.
+Those buffers are only used, if despite presence of a backpressure, Flink can not stop producing more records to the output.
```

Comment:
Hmmm, I've rephrased it to this:
> Those buffers are only used, if the subtask is backpressured 
> by downstream subtasks but the subtask can not gracefully pause its current process. This can happen
> in situations like:


From your proposal I don't like:
> the subtask still needs to produce more records to the output.

as it can be easily confused with regular processing of records from the input and producing a single output record.

Reply:
Thanks @pnowojski quick fix!

Other comments are good to me, but I still have a question about this comment.

I think `but the subtask can not gracefully pause its current process.` is too technical. For some flink users, they may be curious: what is the `gracefully pause its current process`? 

If `it can be easily confused with regular processing of records from the input and producing a single output record.`, could we explain it clearly? Make it clear to users that overdraft buffers address scenarios when processing a single record may require multiple network buffers. Then we can list some situations, it's your  ` This can happen in situations like:`

Reply:
What about:
> Those buffers are only used, if the subtask is backpressured
> by downstream subtasks and the subtask requires more than a single network buffer to finish what its
> currently doing. This can happen in situations like:

?

Reply:
LGTM